src/relay/backend/contrib/ethosu/to_te_graph.cc

```diff
@@ -0,0 +1,234 @@
+/*
```

Comment:
Is it possible for us to combine this with the current code we factored out as part of the `te_compiler.cc` refactor? Ok to do in follow up.